### PR TITLE
{math}[GCCcore/7.3.0] CoinUtils v2.11.3

### PR DESCRIPTION
**Pull request by tstrempel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/CoinUtils/CoinUtils-2.11.3-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+builddependencies = [
+    ('binutils', '2.30'),
+    ('Doxygen', '1.8.14'),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['include', 'lib'],
```
Comment (boegel) [inline easybuild/easyconfigs/c/CoinUtils/CoinUtils-2.11.3-GCCcore-7.3.0.eb]: @tstrempel It's helpful to make this a bit more specific, I would use this:

```python
sanity_check_paths = {
    'files': ['libCoinUtils.%s' % SHLIB_EXT],
    'dirs': ['include/coin', 'lib/pkgconfig'],
}
```

Comment (tstrempel) [reply easybuild/easyconfigs/c/CoinUtils/CoinUtils-2.11.3-GCCcore-7.3.0.eb]: I changed it to this: 
```
sanity_check_paths = {
    'files': ['lib/libCoinUtils.%s' % SHLIB_EXT],
    'dirs': ['include/coin', 'lib/pkgconfig'],
}
```

Without the `lib/` it threw an error on my end, but all of the checks passed...

Comment (boegel) [reply easybuild/easyconfigs/c/CoinUtils/CoinUtils-2.11.3-GCCcore-7.3.0.eb]: Not including the `lib/` was my mistake of course.

The tests run in GitHub don't actually try to install the software, so they wouldn't know whether or not the sanity check paths are correct.

We do a test build on systems where we have sufficient resources to make sure the easyconfig file is correct, I'll look into this right now and get this merged, thanks a lot!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3152.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/551a7661503267fa459711933831e574 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/c7c1a7ac6a8f8db0e5bf645dbaea33ce for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2412.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/bf637d71e65a4590af8d99d15033c87f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @tstrempel!

**Pull request merged**